setup.py

```diff
@@ -28,6 +28,11 @@
     reqs = [r for r in reqs if not r.strip().startswith("#")]
 
 
+with open("requirements-dev.txt", "r") as fd:
```

Comment:
On a second thought, we don't really need to changes this file.
Let's just explicitly mention that the contributors should use `requirements-dev.txt` for installing dependencies and keep [`setup.py`](https://github.com/facebookincubator/python-nubia/blob/master/setup.py) as is.